src/components/TablePreview/TablePreview.tsx

```diff
@@ -32,29 +40,32 @@ const TablePreview: FC<Props> = ({}) => {
 
     return (
         <Flex direction="column" height="100%" alignItems="center" marginBottom="size-1200">
-            <table ref={tableRef}>
-                <tbody>
-                    <tr>
-                        <td rowSpan={3} align="center">
-                            <img src={imgUrl} height="60"></img>
-                        </td>
-                        <td style={name.style}>{name.textValue}</td>
-                        <td style={spacerStyle}></td>
-                        <td style={phone.style}>{phone.textValue}</td>
-                    </tr>
-                    <tr>
-                        <td style={title.style}>{title.textValue}</td>
-                        <td style={spacerStyle}></td>
-                        <td style={email.style}>{email.textValue} </td>
-                    </tr>
-                    <tr>
-                        <td style={company.style}>{company.textValue}</td>
-                        <td style={spacerStyle}></td>
-                        <td style={linkedin.style}>{linkedin.textValue}</td>
-                    </tr>
-                </tbody>
-            </table>
-
+            {hasData ? (
```

Comment:
Lovely ternary!

Reply:
Would be nice to include the copy button within the ternary? Currently the button appears even when there's no signature

<img width="373" alt="image" src="https://github.com/evargast/signature-generator/assets/52969178/dd4a9004-0f10-4ccb-ae94-b515cbe08608">

Reply:
ok ok placing button within the ternary to render with table and the View component if there is data..